client/src/pages/ProfilePages/EditProfile/EditForm/EditForm.tsx

```diff
@@ -1,28 +1,295 @@
-import { Formik, Form } from 'formik';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { useEffect, useState } from 'react';
 import Card from '@mui/material/Card';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import FormGroup from '@mui/material/FormGroup';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import { object, string } from 'yup';
 import useStyles from './useStyles';
 
 const initialValues = {
-  firstname: '',
-  lastname: '',
-  gender: '',
-  birthdate: '',
-  email: '',
+  firstname: 'John',
+  lastname: 'Doe',
+  gender: 'male',
+  birthMonth: 'june',
+  birthDay: '15',
+  birthYear: '1988',
+  email: 'john-doe@gmail.com',
   phone: '',
   address: '',
   description: '',
 };
 
+function TextError(props: any) {
+  const classes = useStyles();
+  return <div className={classes.errors}>{props.children}</div>;
+}
+
+export interface Phone {
+  phone: boolean;
+}
+
 export default function EditForm() {
   const classes = useStyles();
-  const handleSubmit = () => {
-    console.log('submitted');
+  const [isPhone, setIsphone] = useState(false);
+  const [values, setValues] = useState(initialValues);
+
+  initialValues.phone ? setIsphone(true) : null;
```

Comment:
Initial values will never have a phone number, I don't understand the purpose of this line? :)